README.md

```diff
@@ -132,7 +132,8 @@ To set options and keep lint-staged extensible, advanced format can be used. Thi
 
 ## Filtering files
 
-It is possible to run linters for certain paths only by using [minimatch](https://github.com/isaacs/minimatch) patterns. The paths used for filtering via minimatch are relative to the directory that contains the `.git` directory. The paths passed to the linters are absolute to avoid confusion in case they're executed with a different working directory, as would be the case when using the `gitDir` option.
+It is possible to run linters for certain paths only by using [minimatch](https://github.com/isaacs/minimatch) patterns.
+The file patterns should be relative to the project root(where `lint-staged` is installed).
```

Comment:
Why break into multiple lines?

Reply:
Same question. Let's keep it in one paragraph.

Reply:
Having it over multiple lines provides better clarity on what was actually changed when anything is modified in the paragraph. Otherwise it'll just be one line removed and one line added, very difficult to see what changed.

Reply:
Let me know if you'd still prefer I change it to a single line. Either way it wouldn't make any difference to the way it is rendered.